controllers/app.controller.js

```diff
@@ -12,10 +12,17 @@ exports.getEndpoints = (req, res, next) => {
   });
 };
 
-exports.getArticles = (req, res, next) => {
+exports.getArticlesByID = (req, res, next) => {
   const { article_id: id } = req.params
-  selectArticles(id).then((articles)=>[
+  selectArticles(id).then((articles)=>{
     res.status(200).send(articles)
-  ])
+})
   .catch(next)
 };
+
+exports.getArticlesByQuery = (req, res, next) => {
```

Comment:
i think this name is a little confusing because you're not using queries for this endpoint?